notebooks/working/5.0-mab-simulation-causal-graph.md

```diff
@@ -1840,22 +1885,20 @@ for sim_size, number in zip(simulation_sizes, sim_number):
 # 5. Plotting results of the simulation 
 
 ```python
-df = pd.concat([models_dictionary[number].coefs for number in models_dictionary.keys()], axis=1).T
-df.reset_index(drop=True,inplace=True)
+# Creating a dataframe for model parameters
+sim_estimated_par = pd.concat([models_dictionary[number].coefs for number in models_dictionary.keys()], axis=1).T
+sim_estimated_par.reset_index(drop=True,inplace=True)
+sim_estimated_par.head()
 ```
 
 ```python
-def Plot(sim_par, model, size):
-    bins = np.linspace(sim_par.min(), sim_par.max(), 10)
-    plt.figure(figsize=size)
-    sns.distplot(sim_par, kde=False, bins=bins, label='Simulated Parameters')
-    plt.axvline(model.coefs[sim_par.name], color='black', ls='--',label='Data Generating Parameter')
-    plt.title(label='Histogram of '+'"'+sim_par.name+'"', fontdict={'fontsize':16})
-    plt.ylabel('Frequency', rotation=0, labelpad=50)
-    plt.legend()
+# Loop around the different parameters
+# and plot the figures:
+
+for par in sim_estimated_par.columns:
+    PlotParams(sim_estimated_par[par],mnl_model, (10,6))
```

Comment:
Beautiful. These plots are exactly what I was looking for and the helper function to plot them is much appreciated!